// jukebox-receiver.h : Include file for standard system include files,
// or project specific include files.

#pragma once

#include <iostream>
#include "jukebox.pb.h"
#include <google/protobuf/util/delimited_message_util.h>
#include <boost/asio.hpp>
#include <portaudio.h>

// TODO: Reference additional headers your program requires here.
